### ISSUE TITLE
Incompatible with Flags extension

### ISSUE THREAD

**Issue by hrvoje-hr** [club-1/flarum-ext-cross-references#5]
This extension is incompatible with the default Flags extension. Tested in the production forum and on the freshly installed instance as well.

When a post is flagged, all remaining posts in that discussion are invisible and the Scrubber is unusable.

### With Cross-references enabled:
<img width="870" alt="2022-12-15 12_01_43-asdad - Testni flarum" src="https://user-images.githubusercontent.com/58295114/207843118-f71b71a5-f177-46e5-ad92-e9612c1661f6.png">

### Disabled:
<img width="829" alt="2022-12-15 12_02_26-asdad - Testni flarum" src="https://user-images.githubusercontent.com/58295114/207843123-4b10d2b7-3690-4635-8480-e8e34f4e0356.png">

**JS Console errors:**

```
mount-redraw.js:15 TypeError: Cannot read properties of undefined (reading 'view')
    at render.js:141:24
    at render.js:160:3
    at c (render.js:66:8)
    at l (render.js:50:5)
    at render.js:134:5
    at c (render.js:63:14)
    at l (render.js:50:5)
    at render.js:134:5
    at c (render.js:63:14)
    at l (render.js:50:5)
```

**Comment by n-peugnet** [comment]
Thanks, in fact this is already fixed in master by 6c9583402821bcedc17ab04c69e3a925112d804b

I will make a release right now with this change!

**Comment by n-peugnet** [comment]
Released in [v0.4.0](https://packagist.org/packages/club-1/flarum-ext-cross-references#v0.4.0) (it may not be downloadable right away with composer)
